source/guide_rocketchat.rst

```diff
@@ -0,0 +1,305 @@
+.. highlight:: console
+
+.. author:: Dustin Skoracki <https://github.com/d-sko>
+
+.. tag:: lang-nodejs
+.. tag:: web
+.. tag:: chat
+
+.. sidebar:: About
+
+  .. image:: _static/images/rocketchat.svg
+      :align: center
+
+###########
+Rocket.Chat
+###########
+
+.. tag_list::
+
+Rocket.Chat_ is an self hostable, open source chat software written in JavaScript.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+  * :manual:`web backends <web-backends>`
+
+License
+=======
+
+All relevant legal information can be found here
+
+  * https://github.com/RocketChat/Rocket.Chat/blob/master/LICENSE
+
+Prerequisites
+=============
+
+We're using :manual:`Node.js <lang-nodejs>` in the stable version 12:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '12'
+ [isabell@stardust ~]$
+
+We'll also need :lab:`MongoDB <guide_mongodb>`, so follow the MongoDB guide and come back when it's running.
```

Comment:
How can we feature this more prominently? Maybe add it to the "familiar with the basic concepts of"-list? put it into a note box? List it as the first (or last) prerequisite?